### DISCUSSION TITLE
I do not have the 2FA authentication app and the github-recovery-codes.txt. I am unable to log in to GitHub.

### DISCUSSION THREAD

**Discussion by yang20711** [community/community#144989] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I am currently completely unable to log in to GitHub. My phone is broken, and besides having my username and password, I have no other information! Could you please advise how to handle this?

**Comment by davevad93** [comment]
Hi @yang20711 ,

I suggest you to contact support by opening a ticket [here](https://support.github.com/contact/account), select `Sign in, verification, or other account questions` and explain your issue.
